internal/research/opentelemetry/instrumentation-metrics.md

```diff
@@ -0,0 +1,50 @@
+# OpenTelemetry SDK Usage for Metrics in Golang Applications
+
+OpenTelemetry is a project to handle telemetry data (logs, metrics and traces) in a vendor-neutral way. It provides SDKs to instrument applications, APIs and processing tools (collector).
```

Comment:
```suggestion
OpenTelemetry is a project to handle telemetry data (logs, metrics and traces) in a vendor-neutral way. It provides SDKs to instrument applications, APIs and processing tools ([Collector](https://opentelemetry.io/docs/collector/)).
```

I'm getting the suspicion that "collector" here was a generic word? Are collectors a kind of processing tools? Would it be more correct to say "...processing tools (called 'collectors')."